plugin/pkg/admission/noderestriction/admission_test.go

```diff
@@ -24,7 +24,6 @@ import (
 	"testing"
 	"time"
 
-	corev1 "k8s.io/api/core/v1"
```

Comment:
prefer corev1

Reply:
Agree - this is pretty consistent across codebase